src/App.js

```diff
@@ -1,9 +1,21 @@
-import "./App.css";
+import {
+  CartManagement,
+  ProductListing,
+  LandingPage,
+  Wishlist,
+  Login,
+  Signup,
+} from "./pages/index.js";
```

Comment:
If you are importing all the files from pages/index.js then you can ommit index.js here it will still work.

Reply:
I didnt get what yu are saying , you explain again